.github/workflows/functional.yml

```diff
@@ -104,6 +106,12 @@ jobs:
           SECRETS_RSTUF_TOKEN_KEY: test-token-key
           SECRETS_RSTUF_ADMIN_PASSWORD: secret
     steps:
+      - name: Get RSTUF Container logs
+        run: |
+          docker logs "${{ job.services.rstuf-worker-1.id }}"
```

Comment:
The `${{ job.services.rstuf-worker-1.id }}` would give the id of the container right?